index.html

```diff
@@ -60,7 +60,7 @@ <h1>WebRTC samples</h1>
 
       <p>This is a repository for the WebRTC Javascript code samples. The source for these samples is available at <a href="https://github.com/webrtc/samples" title="View GitHub repository for these files">github.com/webrtc/samples</a>.</p>
 
-      <p>Some of the samples use new browser features. They may only work in <a href="https://www.google.com/chrome/browser/canary.html" title="Download Chrome Canary">Chrome Canary</a>, <a href="http://www.mozilla.org/firefox/beta/" title="Download Firefox Beta">Firefox Beta</a> or Microsoft Edge (available with Windows 10), and may require flags to be set.</p>
+      <p>Some of the samples use new browser features. They may only work in <a href="https://www.google.com/chrome/browser/canary.html" title="Download Chrome Canary">Chrome Canary</a>, <a href="http://www.mozilla.org/firefox/beta/" title="Download Firefox Beta">Firefox Beta</a>, Microsoft Edge (available with Windows 10) or Safari (supporting WebRTC since High Sierra) and may require flags to be set.</p>
```

Comment:
> High Sierra

What about iOS?

Reply:
Should refer to Safari Tech Preview (https://developer.apple.com/safari/technology-preview) and Microsoft Edge Preview (https://insider.windows.com).